src/components/map/hooks/use-control-styles.ts

```diff
@@ -29,8 +29,7 @@ export function useControlPositionsStyle() {
       },
       controlsContainer: {
         position: 'absolute',
-        bottom:
-          bottom + bottomPaddingIfBottomSheetIsOpen + theme.spacings.medium,
+        bottom: bottomPaddingIfBottomSheetIsOpen + theme.spacings.medium,
```

Comment:
@tormoseng @hanne-at-atb This is the change we need to check if also works for Android phones with the Android NavBar covering the bottom of the screen.

Reply:
Update: turns out it works fine